extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/ReactiveClientHeadersFactory.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.rest.client.reactive;
+
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.eclipse.microprofile.rest.client.ext.ClientHeadersFactory;
+
+import io.smallrye.mutiny.Uni;
+
+/**
+ * Reactive ClientHeadersFactory flavor for Quarkus rest-client reactive extension.
+ */
+public abstract class ReactiveClientHeadersFactory implements ClientHeadersFactory {
```

Comment:
do we need to extend `ClientHeadersFactory`?

Reply:
Yes, to be used with `@RegisterClientHeaders`.